docs/okeps/okep-5212-ovnobserv-api.md

```
@@ -0,0 +1,241 @@
+	// +kubebuilder:validation:Maximum=100
+	// +required
+	Probability int32
+	// Feature is the Observability feature that should be sampled.
+	// +kubebuilder:validation:Required
+	// +required
+	Feature ObservabilityFeature
+}
+
+// ObservabilityStatus contains the observed status of the ObservabilityConfig.
+type ObservabilityStatus struct {
+	// +listType=map
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)